### PR TITLE
Send credentials on requests to api

### PR DESCRIPTION
**Pull request by JackWilb**
Also fixed the string interpolation in this file

### CODE REVIEW

File: src/lib/multinet.ts [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-async function _loadLinks(workspace: string, edgeTable: string, apiRoot: string) {
-  const linksCall = apiRoot + '/workspaces/' + workspace + '/tables/' + edgeTable + '?limit=1000';
-  const linksRaw = await json(linksCall);
-  return linksRaw.rows;
+  // Else if the table is small enough, loop through the rows and download
+  let output: any[] = [];
+  output = output.concat(table.rows);
+
+  while (output.length < table.count) {
+    table  = await api.table(workspace, tableName, { offset: output.length, limit: 30 });
```
Comment (waxlamp) [inline src/lib/multinet.ts:21]: You can probably up this to 100 rather than 30.

File: src/lib/multinet.ts [hunk 2]
```diff
@@ -42,35 +49,37 @@ function _renameNodeVars(nodes: any[]) {
...
-  // Loop through each node tables and fetch the nodes to global variables
-  for (const nodeTable of multinet.tables.nodeTables) {
-    const ntable = await _loadNodes(workspace, nodeTable, apiRoot);
-    multinet.nodes = multinet.nodes.concat(ntable);
+  // Loop through each node tables and fetch the nodes
+  for (const tableName of multinet.tables.nodeTables) {
+    multinet.nodes = multinet.nodes.concat(await _downloadAllRows(api, workspace, tableName, 'node'));
   }
 
   // Load the edge table (ONLY ONE BECAUSE OF ARANGO API LIMITATIONS)
```
Comment (waxlamp) [inline src/lib/multinet.ts:81]: This comment is a tiny bit misleading--it does originate from Arango's setup, but a single edge table is now part of the Multinet API as well. It's not a huge deal but I'd remove the parenthesis from this comment.

File: src/lib/multinet.ts (hunk 1 above)
Comment (JackWilb) [reply src/lib/multinet.ts]: Fixed in 69b106e

File: src/lib/multinet.ts (hunk 2 above)
Comment (JackWilb) [reply src/lib/multinet.ts]: Fixed in 69b106e

**Pull request merged**